README.md

```diff
@@ -1,6 +1,91 @@
 # mentorship-page [![Netlify Status](https://api.netlify.com/api/v1/badges/f804919f-a599-456c-8127-496ed860123c/deploy-status)](https://app.netlify.com/sites/tender-meitner-99286b/deploys)
 
-React app for Advisory's Mentorship Network page. This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+React app for Advisory's Mentorship Network page.
+
+## Project Architecture
+
+```
+[ Airtable Mentor Entries ]
+            ▼
+ [ AWS Lambda Function ]
+            ▼
+    [ ElasticSearch ]
+            ▼
+    [ React Web App ]
+```
+
+Airtable is a database we use to easily store our mentor data, such as
+Mentor name, images, educational background, and more.
+
+AWS Lambda is a compute service which allows us to run custom code on the
+cloud without managing infrastructure. We can run our AWS Lambda script
+periodically to read the Airtable mentor entries using the API, filter
+them out, and update the ElasticSearch service with our Airtable Mentor Data.
+
+ElasticSearch is an open source search and analytics engine. It allows
+us to store, search and analyse huge volumes of mentor data quickly and
+in near real-time and give back answers in milliseconds. We hosted an
+ElasticSearch instance on Elastic Cloud.
+
+Our React Web App queries the ElasticSearch Service and displays the mentors.
+A search query is sent from the web app and is used to retrieve the relevant
+mentors.
+
+Our React Web App is hosted on Netlify. Netlify also has deploy previews
+which are created for each pull request, which allows us to easily test
+our code changes
+
+## Getting Started
+
+To get started, make sure NodeJS is installed on your computer.
+You can install it following the instructions [here](https://nodejs.org/en/)
+
+Afterwards, we can download this repo onto our computer, or we can clone
+this repo using the command
+`git clone https://github.com/AdvisorySG/mentorship-page.git`
```

Comment:
This should be block code for ease of copying.

Also, please specify the `$` sign infront of the commands (following usual Bash command prompt conventions)